src/components/Reviews/ReviewCard.jsx

```diff
@@ -0,0 +1,54 @@
+import { ReactComponent as CommentIcon } from "../../icons/comment.svg";
+import { ReactComponent as VotesIcon } from "../../icons/votes.svg";
+
+const ReviewCard = ({ review }) => {
+  const {
+    title,
+    category,
+    review_img_url,
+    owner,
+    review_body,
+    comment_count,
+    votes,
+    created_at,
+  } = review;
+  const getReviewTextPreview = () => {
+    const firstSentenceOfReview = review_body.match(/^.*?[\.!\?](?:\s|$)/)[0];
```

Comment:
This is a cool solution to limit the preview text length 👍 It might be nice to break this out to a seperate utils file just to keep the component as clear as possible